### DISCUSSION TITLE
A variety of people and portraits and places

### DISCUSSION THREAD

**Discussion by Triceratops9** [TheLastBen/fast-stable-diffusion#1055] [General]
Hi there,
I'd like to use Dreambooth to create many portraits of people who don't really exist using photographic styles I can build myself. If I'd like to generate a diverse community of portraits, how many images should I upload as instance photos? What is the best number of images to recreate a large and diverse selection of final images ?(age, skin tone, location, etc.) 

For my concept images, how many images should be uploaded to help define a style etc.

Thank you!

**Comment by TheLastBen** [comment]
for concept images, you can go as high as 600, but 200 is fine, make sure they have the same style as the instance images, as for the instance images, try 20, and set the text_encoder to 350, the concept steps to 1500

**Reply by Triceratops9** [reply]
Thank you! One more total newbie question (bear with  me.) How do I actually get my images into into Dreambooth? I attached screen.  shots for how I included the path to my google drive but I'm getting errors. 

![Constant Dreamstudio Screen Shot](https://user-images.githubusercontent.com/118769818/208242232-9476a063-1eaf-4e4e-8a5f-fd8b0b5cbb54.jpg)
![Dreambooth instance Screen shot](https://user-images.githubusercontent.com/118769818/208242234-37debc95-c375-4b44-993a-509432deddce.jpg)

**Reply by TheLastBen** [reply]
in the first picture, you should input a path to your gdrive like what you did in the second picture.

in the second picture, the folder name should not contain spaces.

**Reply by Triceratops9** [reply]
Thank you! I just tried everything again. I'm getting an error when I create a session though it's letting me go on. Then i'm getting another error when I try to load instance images. Screenshot attached. 
![Screenshot 2022-12-17 125826](https://user-images.githubusercontent.com/118769818/208255236-17096569-29f6-44ce-9c08-e9786bd4b350.jpg)

![Screenshot 2022-12-17 125847](https://user-images.githubusercontent.com/118769818/208255238-79422ccd-1389-4cad-bcfe-31f398e9f0fd.jpg)

**Reply by TheLastBen** [reply]
problem with browser cookies, clear them or use a different browser

**Reply by Triceratops9** [reply]
That worked! Thank you!
I'm close-- When I'm training, the training stops because I get disconnected. This has happened twice so far. What do I need to do so that doesn't happen?
Thanks so much for all your help.

**Reply by TheLastBen** [reply]
save intermediary checkpoints, and in case you get disconnected, resume the session and load the latest saved model
